marketo_ma.install

```diff
@@ -5,7 +5,68 @@
  * Install hooks for Marketo MA module.
  */
 
-use Drupal\Core\Link;
+/**
+ * Implements hook_schema().
```

Comment:
I'll offer a little feedback here. Custom tables aren't used in Drupal 8 as often as they are in d7. There are a couple of reasons but the most cited reason is that you have to maintain the schema which is a pain point sometimes. Typically if you need to create a custom db schema, you kinda need an entity in which case you get views support for free so most people go that route. This data can be considered a persistent cache. There is two ways to create one of those in drupal. 

1. Using the state api (previous implementation). 
2. A custom key/value collection `\Drupal::keyValue('collection_name')`. 

The key/value collection is probably the applicable as state is supposed to be reserved for data that "Cannot be recreated" such as the last cron run time.

Getting slightly more philosophical, this cached field data might really belong in cache. If a very long cache time was set it would be fine in most cases.